docs/browser.md

```diff
@@ -65,11 +71,11 @@ the following changes:
 --- a/hardhat-boilerplate/frontend/src/components/Dapp.js
 +++ b/hardhat-boilerplate/frontend/src/components/Dapp.js
 @@ -2,6 +2,7 @@
- 
+
  // We'll use ethers to interact with the Ethereum network and our contract
  import { ethers } from "ethers";
```

Comment:
To try to minimize the bundle. I would try to avoid this import, which is probably the reason why the whole library is included. I did look at ethers-v6, and it looks like they have ESM build, which should enable tree shaking.

What I suggest, if we are only using only JsonRpcProvider, to import it like this:
```typescript
import { JsonRpcProvider } from 'ethers/providers'
```

And see if it affects the bundle. I am not sure if you can do even more specific imports, so that not all providers all imported.

Reply:
Ah, my bad, this is in `.md`. I meant in our code 😅

Reply:
Ethers does support tree shaking, and in the sapphire-paratime package we're importing only specific functions/classes.

The only improvement that can be made is removing the `wrap` function in favor of `wrapEthersSigner` and `wrapEthersProvider` etc.

There are a few places where Ethers is being used lazily where an EIP-1193 compatible provider could be used directly. Combine that with removing `wrap` and tree-shaking will be better able to exclude unused code.

Reply:
Not sure, why https://github.com/oasisprotocol/sapphire-paratime/pull/292#issuecomment-2016555236 the ethers chunk is like 1/3th of the bundle. But ok I guess. I just wanted to point out, we could try with more specific imports, to see if it optimizes the bundle.

Reply:
Yup I'm very aware that it's a large part of the bundle, but I won't be able to address that in this PR without making some major changes to sapphire-paratime package which requires introducing a sapphire-ethers-v6 package.